boule/_triaxialellipsoid.py

```diff
@@ -22,8 +22,10 @@ class TriaxialEllipsoid:
 
     The ellipsoid is defined by five parameters: semimajor axis, semimedium
     axis, semiminor axis, geocentric gravitational constant, and angular
-    velocity The thee semi-axis are different and the ellipsoid spins around
-    it's largest moment of inertia.
+    velocity. The ellipsoid spins around it's smallest semiminor axis, which
```

Comment:
```suggestion
    velocity. The ellipsoid spins around its smallest semiminor axis, which
```

Reply:
thanks. I made the change and also reworded what follows slightly.